docs/tool-calling.md

```diff
@@ -214,7 +214,7 @@ const getWeather = ai.defineTool(
 );
 
 const generateOptions: GenerateOptions = {
-  prompt: 'What's the weather like in Baltimore?',
+  prompt: "What's the weather like in Baltimore?",
```

Comment:
consistency -- single quote are used above.

Reply:
but the apostrophe

Reply:
ah, all good then!